app/scripts/controllers/network/network-controller.test.js

```diff
@@ -1,7 +1,56 @@
 import sinon from 'sinon';
+import nock from 'nock';
 import { getNetworkDisplayName } from './util';
 import NetworkController, { NETWORK_EVENTS } from './network-controller';
 
+/**
+ * Construct a successful RPC response.
+ *
+ * @param {any} result - The RPC result to return.
+ * @returns A successful RPC response with the specified result.
+ */
+function constructSuccessfulRpcResponse(result) {
+  return {
+    id: 1,
+    jsonrpc: '2.0',
+    result,
+  };
+}
+
+// Example block taken from Ethereum Mainnet
+const BLOCK = {
```

Comment:
Does this need to contain a complete set of data for a block? I assume so but just want to check.

Reply:
I don' think so. I made it complete just in case. I'll be reusing this as the basis for additional mocks to be added in a later PR as well.